pubsub/cloud-client/subscriber.py

```diff
@@ -89,6 +89,35 @@ def delete_subscription(project, subscription_name):
     print('Subscription deleted: {}'.format(subscription_path))
 
 
+def update_subscription(project, subscription_name, ack_deadline_seconds):
+    """
+    Updates an existing Pub/Sub subscription's ackDeadlineSeconds
+    from 10 seconds (default). Note that certain properties of a
+    subscription, such as its topic, are not modifiable.
+    """
+    subscriber = pubsub_v1.SubscriberClient()
+    subscription_path = subscriber.subscription_path(
+        project, subscription_name)
+
+    subscription = {
```

Comment:
in the other samples we actually use the real types over dictionaries, so maybe make this `subscription = pubsub_v1.types.Subscription(name=..., ack_deadline_seconds=...)`?

Reply:
Done.